tests/graphql/operations/jsonplaceholder.graphql

```diff
@@ -0,0 +1,46 @@
+#> server-sdl
+schema
+  @server(port: 8000, graphiql: true, hostname: "0.0.0.0")
+  @upstream(baseURL: "http://jsonplaceholder.typicode.com", httpCache: true) {
+  query: Query
+}
+
+type Query {
+  posts: [Post] @http(path: "/posts")
+  user(id: Int!): User @http(path: "/users/{{args.id}}")
+}
+
+type User {
+  id: Int!
+  name: String!
+  username: String!
+  email: String!
+  phone: String
+  website: String
+}
+
+type Post {
+  id: Int!
+  userId: Int!
+  title: String!
+  body: String!
+  user: User @http(path: "/users/{{value.userId}}")
+}
+
+#> client-query
+query @diagnostic(count: 0) {
```

Comment:
could we borrow the approach to test errors from the graphql tests like here https://github.com/tailcallhq/tailcall/blob/0295e5e16a9c13aa93864b155c3857090e9707a5/tests/graphql/errors/test-field-already-implemented-from-Interface.graphql#L21 ?

- more implicit naming for directive and arguments
- ability to test actual error